config/sidekiq.yml

```diff
@@ -10,7 +10,6 @@
   - bulk_republishing
   - sync_checks
   - link_checks
-  - asset_migration
```

Comment:
`lib/migrate_assets_to_asset_manager.r` still uses the `asset_migration` queue.  I think if the queue is being removed, that should be too.

Reply:
Good catch, I've put the queue back though as I don't want to delete too much in one PR.